CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,265 @@
+# Release notes for v2.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.0.4
+
+## Changes by Kind
+
+### API Change
+
+- -capacity-controller-deployment-mode gets replaced with -enable-capacity ([#540](https://github.com/kubernetes-csi/external-provisioner/pull/540), [@pohly](https://github.com/pohly))
+
+### Feature
+
+- Added leader election health check at the metrics port + path "/healthz/leader-election".
+  - klog/v2 is used for logging
+  - process_start_time metric is now reported
+  - `metrics-address` flag is deprecated and replaced by `http-endpoint`, which enables handlers from both metrics manager and leader election health check. ([#537](https://github.com/kubernetes-csi/external-provisioner/pull/537), [@verult](https://github.com/verult))
+- External-provisioner can be deployed alongside a CSI driver on each node to manage local volumes. ([#524](https://github.com/kubernetes-csi/external-provisioner/pull/524), [@pohly](https://github.com/pohly))
+- Add `--immediate-topology` to controls what topology information is passed to `CreateVolumeRequest.AccessibilityRequirements` ([#501](https://github.com/kubernetes-csi/external-provisioner/pull/501), [@pohly](https://github.com/pohly))
```

Comment:
@msau42 can you also review this? There is no release-notes in that PR so I just extract this from the flag description.

Reply:
maybe something like "Add --immediate-topology to toggle if topology is passed to CreateVolumeRequest for immediate binding"